### PR TITLE
DB-9428 DB Pipeline to cloud

### PR DESCRIPTION
**Pull request by bklo94**

### CODE REVIEW

File: pipelines/k8s-test/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    }
+    stage('Run Integration Tests') {
+        wrap([$class: 'VaultBuildWrapper', vaultSecrets: vaultSecrets]) {
+            dir ('dbaas-infrastructure/kubernetes/') {
+                checkSpliceDB()
+                def jdbc_ip = sh (script:"kubectl get svc -A | grep 'LoadBalancer.*1527' | awk '{\$1=\$1};1' | cut -d \" \" -f 5 | xargs echo -n", returnStdout: true)
+                if ( jdbc_ip == "") {
+                    throw exception("JDBC URL not found/initialized")
+                }
+                sh "helm install --name integration-test charts/integration-tests --namespace test --set jdbc_url='jdbc:splice://$jdbc_ip:1527/splicedb;ssl=basic;password=admin;' --set splice_release='DB-9133_3.0' --set artifact_user='$jenkins_user' --set artifact_password='$jenkins_pass'"
```
Comment (martinrupp) [inline pipelines/k8s-test/Jenkinsfile:143]: DB-9133-3.0? that should maybe be a variable, no?

File: pipelines/k8s-test/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+                [$class: 'StringParameterValue', name: 'tag_creator', value: scmVars.GIT_BRANCH]
+            ]
+            }
+        }
+        notifyBuild(currentBuild.result)
+}
+
+//scheck for external IP to show and if ingress is accepting connections
+def checkSpliceDB() {
+    sh '''wget http://sourceforge.net/projects/netcat/files/netcat/0.7.1/netcat-0.7.1.tar.gz
```
Comment (martinrupp) [inline pipelines/k8s-test/Jenkinsfile:195]: shouldn't this be defined in the requirements e.g. Dockerfile?
`RUN apt-get update && apt-get install -y netcat`

File: pipelines/k8s-test/Jenkinsfile (hunk 1 above)
Comment (bklo94) [reply pipelines/k8s-test/Jenkinsfile]: Updated it to set the release to the current branch that its pulling from. At the time, DB-9133 was the only branch that had working remote-it's.

File: pipelines/k8s-test/Jenkinsfile (hunk 2 above)
Comment (bklo94) [reply pipelines/k8s-test/Jenkinsfile:173]: I ran into errors where netcat wasn't found in the docker image. I tried putting the package into the docker image as an `apk install`(base is alpine), but I kept getting an error about application not found. I also tried exporting the path inside the docker image without success. The only way I managed to get it working was just to install it as a part of the pipeline as a stage.

Comment (martinrupp) [reply pipelines/k8s-test/Jenkinsfile:173]: ok

File: pipelines/k8s-test/Jenkinsfile (hunk 1 above)
Comment (martinrupp) [reply pipelines/k8s-test/Jenkinsfile]: 👍

**Review by martinrupp — COMMENTED** [review]
looks good to me

**Review by martinrupp — APPROVED** [review]

**Pull request merged**